packages/apple-pay-integration/e2e/ApplePaySessionCustomerStepMockObject.ts

```diff
@@ -0,0 +1,180 @@
+                                }
+                            },
+                            transactionIdentifier: "xx",
+                        },
+                        billingContact: {
+                            emailAddress: "mock@mock.com",
+                            familyName: "mock",
+                            givenName: "mock",
+                            phoneNumber: "00000000",
+                        },
+                        shippingContact: {
+                            emailAddress: "mock@mock.com",
+                            familyName: "mock",
+                            givenName: "mock",
+                            phoneNumber: "00000000",
+                        }
+                    },
+                    bubbles: false,
+                    cancelBubble: false,
+                    cancelable: false,
+                    composed: false,
+                    currentTarget: {
+                        addEventListener: () => { return null; },
+                        dispatchEvent: () => { return true },
+                        removeEventListener: () => { return null; },
+                    },
+                    defaultPrevented: false,
+                    eventPhase: 1,
+                    isTrusted: false,
+                    returnValue: false,
+                    srcElement: {
+                        addEventListener: () => { return null; },
+                        dispatchEvent: () => { return true },
+                        removeEventListener: () => { return null; },
+                    },
+                    target: {
+                        addEventListener: () => { return null; },
+                        dispatchEvent: () => { return true },
+                        removeEventListener: () => { return null; },
+                    },
+                    timeStamp: '',
+                    type: '',
+                    composedPath: () => { return [] },
+                    initEvent: () => { return null; },
+                    preventDefault: () => { return null; },
+                    stopImmediatePropagation: () => { return null; },
+                    stopPropagation: () => { return null; },
+                    AT_TARGET: 0,
+                    BLUR: 0,
+                    BUBBLING_PHASE: 0,
+                    CAPTURING_PHASE: 0,
+                    CHANGE: 0,
+                    CLICK: 0,
+                    DBLCLICK: 0,
+                    DRAGDROP: 0,
+                    FOCUS: 0,
+                    KEYDOWN: 0,
+                    KEYPRESS: 0,
+                    KEYUP: 0,
+                    MOUSEDOWN: 0,
+                    MOUSEDRAG: 0,
+                    MOUSEMOVE: 0,
+                    MOUSEOUT: 0,
+                    MOUSEOVER: 0,
+                    MOUSEUP: 0,
+                    NONE: 0,
+                    SELECT: 0,
+                } as ApplePayJS.ApplePayPaymentAuthorizedEvent;
```

Comment:
Sorry I initially thought you can do something like:
```ts
class ApplePayPaymentAuthorizedEventMock extends Event implements ApplePayJS.ApplePayPaymentAuthorizedEvent {
    payment: {
        // ...
    }
}
```
But it turns out `ApplePayJS.ApplePayPaymentAuthorizedEvent` isn't typed properly - it has its own definition of `Event` rather than the one from `lib.dom.d.ts`. So in this case, I think it's justified to force cast. Could you please revert back to what you had?

Reply:
reverted back to the old event object

Reply:
Updated